frontend/packages/db-structure/src/operation/operationsSchema.ts

```diff
@@ -1,4 +1,28 @@
 import * as v from 'valibot'
+import { tableSchema } from '../schema/index.js'
+import { PATH_PATTERNS } from './constants.js'
+
+const isTablePath = (input: unknown): boolean => {
+  if (typeof input !== 'string') return false
+  return PATH_PATTERNS.TABLE_BASE.test(input)
+}
+
+const addTableOperationSchema = v.object({
+  op: v.literal('add'),
+  path: v.custom<`/tables/${string}`>(
+    isTablePath,
+    'Path must match the pattern /tables/{tableName}',
+  ),
+  value: tableSchema,
+})
```

Comment:
In the interim, I have decided to strictly define the schema for adding tables, possibly supporting many patterns such as columns, indexes, etc.We'll adjust the strictness once we understand how hard it will be.